editor/src/uuiui/widgets/popup-list/popup-list.tsx

```diff
@@ -106,8 +106,7 @@ const Option = (props: OptionProps<SelectOption>) => {
         {props.data.icon == null ? null : (
           <Icn
             {...props.data.icon}
-            // We override the color prop here so it can switch between dark and light depending on whether its highlighted (to always contrast the background)
-            color={props.isFocused ? 'on-highlight-main' : 'main'}
+            color={'white'}
```

Comment:
Was this change meant to be included?

Reply:
yes, because the popup is always dark mode